university-college-lillebaelt-vancouver.csl

```diff
@@ -270,7 +278,7 @@
   </citation>
   <bibliography et-al-min="7" et-al-use-first="6" second-field-align="flush">
     <layout>
-      <text variable="citation-number" suffix="."/>
+      <text variable="citation-number" suffix=". "/>
```

Comment:
The makeBibliography() function delivers a full list of citations, and a note of the offset. IIRC, partial updates based on the list are handled in Zotero in integration.js.